pkg/apis/eventing/v1alpha1/channel_types.go

```diff
@@ -121,7 +121,7 @@ func (c *Channel) GetGroupVersionKind() schema.GroupVersionKind {
 	return SchemeGroupVersion.WithKind("Channel")
 }
 
-// GetSpec returns the spec of the Channel.
-func (c *Channel) GetSpec() interface{} {
+// GetUntypedSpec returns the spec of the Channel.
+func (c *Channel) GetUntypedSpec() interface{} {
```

Comment:
Can you explain why made this change here ?

Reply:
Thanks for the review @daisy-ycguo 

That's changed because the type in `knative/pkg` is created that way instead of `GetSpec`: https://github.com/knative/pkg/pull/467/files#diff-1d8b79b225dae79ae4f68c9ffd87e926R54

See this comment why it was changed: https://github.com/knative/pkg/pull/467#discussion_r295363743